lib/string.gi

```diff
@@ -1258,9 +1258,121 @@ InstallGlobalFunction(StringOfMemoryAmount, function(m)
     Append(s, units[shift+1]);
     return s;
 end);
-
-        
     
+InstallGlobalFunction(PrintToFormatted, function(stream, s, args...)
```

Comment:
Replace `args` by `data` to match documentation? (Or vice versa, adjust the documentation)